### PR TITLE
metrics: improve ut coverage to 100%

### PR DESCRIPTION
**Pull request by bb7133**
### What problem does this PR solve?
Simply add more unit tests for package `metrics`, improve ut coverage to 100%

### What is changed and how it works?
Add UT

### Check List
Tests

### Unit test

### CODE REVIEW

File: metrics/server_test.go [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+package metrics
+
+import (
+	. "github.com/pingcap/check"
+	"github.com/pingcap/errors"
+	"github.com/pingcap/parser/terror"
+)
+
+func (s *testSuite) TestExecuteErrorToLabel(c *C) {
```
Comment (zimulala) [inline metrics/server_test.go]: Could we put this test to `metrics_test.go` directly?

Comment (bb7133) [reply metrics/server_test.go]: Of course...

**Comment by bb7133** [comment]
@zimulala @xiekeyi98 PTAL, thx!

**Pull request merged**